state/remoteentities.go

```diff
@@ -22,6 +24,7 @@ type remoteEntityDoc struct {
 	SourceModelUUID string `bson:"source-model-uuid"`
 	EntityTag       string `bson:"entity"`
 	Token           string `bson:"token"`
+	Macaroon        string `bson:"macaroon,omitempty"`
```

Comment:
Yeah, we could. Sadly I cargo culted this (storing as json) from model migrations code. Better to be consistent?